pkg/cmd/provisioning/aws/create-iam-roles.go

```diff
@@ -413,6 +415,7 @@ func NewCreateIAMRolesCmd() *cobra.Command {
 	createIAMRolesCmd.MarkPersistentFlagRequired("credentials-requests-dir")
 	createIAMRolesCmd.PersistentFlags().StringVar(&CreateIAMRolesOpts.IdentityProviderARN, "identity-provider-arn", "", "ARN of IAM Identity provider for IAM Role trust relationship (can be created with the 'create identity-provider' sub-command)")
 	createIAMRolesCmd.MarkPersistentFlagRequired("identity-provider-arn")
+	createIAMRolesCmd.PersistentFlags().StringVar(&CreateIAMRolesOpts.Region, "region", "", "AWS region endpoint only required for GovCloud")
```

Comment:
Should we just make it required no matter what, that way AWS, GovCloud, and China should "just work"?

Reply:
there is possibility that someone will ask why we need it for non GovCloud

Reply:
Best thing in my opinion will be to document it and say it is required only for GovCloud (we need to test it first)